### ISSUE TITLE
[Workflow] New lines (\n) results in partially coloured label in image generated from PUML dump

### ISSUE THREAD

**Issue by alexislefebvre** [symfony/symfony#49311]
### Symfony version(s) affected

5.4+

### Description

When I use `\n` in the label of a transition, the PNG file generated from the PUML file shows text with and without the color applied.

<del>Using quotes breaks the generation of the PNG image, but it may be the expected behaviour of the YAML Component.</del>

### How to reproduce

```
                symfony_test:
                    from: [pending, created, scheduled]
                    to: courierconfirmed
                    metadata:
                        color: 'DarkGreen'
                        arrow_color: 'DarkGreen'
                        label: "Lorem ipsum\n(hey)"
```

Here is the result:

> ![image](https://user-images.githubusercontent.com/2071331/217781444-0a4c13e2-2828-453f-bca5-c4213dcf45f8.png)

### Possible Solution

Close the `<font>` tag before `\n` and reopen it after that new line.

### Additional Context

See also #49174

**Comment by stof** [comment]
If you use `\n` in a double-quoted Yaml string, you are still passing an actual newline character to the workflow component as part of the label, not a string `\n`. In YAML, double-quoted strings support escape sequences.

**Comment by alexisLefebvre** [comment]
I will perform some tests once #49174 will be merged.

Status: Needs work

---

Note to myself: this tool can be used to test PUML: https://www.plantuml.com/plantuml/uml/SoWkIImgAStDuSf8pIbGACxEp2j8B4hCBUPoICrB0Oa30000

**Comment by lyrixx** [comment]
It is fixed by #49272, isn't it?

**Comment by alexisLefebvre** [comment]
Not yet, the Symfony dumper add only one `<font>` tag, which results in a broken output when `\n` is used, it gives this result: https://www.plantuml.com/plantuml/uml/HSx12W91283XkqyXSr-wnoJx844CErQ2gs56FNxJGkn9yFiHvsUsoDUgq5JzVLdb8tQ3b5H6Vt2qz80H9MDPWiVi1MiLavIcH7_RxWo3lPecslT7BFNgbhYuUfo2EsrqjZkhUZtya0hCR7syymK0

The `<font>` tag should be closed before the new line and added after: https://www.plantuml.com/plantuml/uml/NSv12i9030NGVKuHPb_SonXw4466DceWJIH6zFYEHIcu2lpt2N-yPujyB0fDrP-dHLvY5qX9PVGRHqiFwALaB5DmltF1MiKafIdHpzO_tM1h3GDj-HvBFRibJguUX-2PLggxJqf7-zShg_fN2uniSr_v1W00

**Comment by alexisLefebvre** [comment]
> It is fixed by #49272, isn't it?

Actually it was a not so pretty way to avoid the issue. I'll change that in the PR.
